themes/StarterTheme/templates/_partials/breadcrumb.tpl

```diff
@@ -0,0 +1,12 @@
+<nav>
+  <ol vocab="http://schema.org/" typeof="BreadcrumbList">
```

Comment:
Seems like RDFa is gaining traction, but microdata is probably better supported, so we've switched back to microdata, thanks!